src/Control/Lens/At.hs

```diff
@@ -454,17 +458,25 @@ instance At (Maybe a) where
   {-# INLINE at #-}
 
 instance At (IntMap a) where
+#if MIN_VERSION_containers(0,5,8)
```

Comment:
Eh... Probably better to leave it as you have it actually. I'll see if there's a ticket for `containers` to add any appropriate `RULES`.